pkg/entities/vulnerability/vulnerability.go

```diff
@@ -60,6 +60,9 @@ type Vulnerability struct {
 	//
 	// For more info see https://github.com/ZupIT/horusec/issues/680
 	VulnHashInvalid string `json:"-" gorm:"-" swaggerignore:"true"`
+
+	SecurityToolVersion string `json:"securityToolVersion" gorm:"Column:security_tool_version" example:"1.2.0"`
+	SecurityToolInfoURI string `json:"securityToolInfoUri" gorm:"Column:security_tool_info_uri" example:"https://horusec.io/"`
```

Comment:
I think that we should ignore this field on Gorm. What do you guys think? @nathanmartinszup @wiliansilvazup @iancardosozup ?

Reply:
I agree with @matheusalcantarazup.